app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -232,32 +232,22 @@
           return;
         }
 
-        var lastClicked = loadFromSessionStorage(sessionStoreLink);
+        var lastClicked = loadFromSessionStorage(sessionStoreLink) || $element.find('.' + activeLinkClass).first().attr('data-position');
 
-        if (lastClicked) {
```

Comment:
should you keep this in? What would happen if the active class couldn't be found either?

Reply:
If the active class isn't found then this entire function has already exited on line 231. The rest only gets called if there's more than one active class found.